go/cmd/dolt/commands/diff.go

```diff
@@ -467,6 +479,41 @@ func maybeResolve(ctx context.Context, dEnv *env.DoltEnv, spec string) (*doltdb.
 	return root, true
 }
 
+var diffSummarySchema = sql.Schema{
+	&sql.Column{Name: "Table name", Type: types.Text, Nullable: false},
+	&sql.Column{Name: "Diff Type", Type: types.Text, Nullable: false},
```

Comment:
(minor/nit) `Diff type` to be consistent with the other columns' capitalization?

```suggestion
	&sql.Column{Name: "Diff type", Type: types.Text, Nullable: false},
```